packages/aws-cdk-lib/aws-rds/test/serverless-cluster.test.ts

```diff
@@ -891,7 +891,7 @@ describe('serverless cluster', () => {
 
     // WHEN
     new ServerlessCluster(stack, 'Database', {
-      engine: DatabaseClusterEngine.AURORA,
+      engine: DatabaseClusterEngine.AURORA_POSTGRESQL,
       parameterGroup: ParameterGroup.fromParameterGroupName(stack, 'ParameterGroup', 'default.aurora-postgresql11'),
     });
```

Comment:
Used AURORA_POSTGRESQL because the 'default.aurora-postgresql11' is specified at parameterGroup.